set.mm

```diff
@@ -661380,13 +661385,85 @@ D Fn ( I ... ( M - 1 ) ) /\
       TVAURIKVCVFVGVDULUNUO $.
   $}
 
+  ${
+    $d R a x $.  $d F a $.  $d M a $.  $d ph a x $.
+    rndrhmcl.r $e |- R = ( N |`s ran F ) $.
+    rndrhmcl.0 $e |- .0. = ( 0g ` N ) $.
+    rndrhmcl.h $e |- ( ph -> F e. ( M RingHom N ) ) $.
+    rndrhmcl.m $e |- ( ph -> M e. DivRing ) $.
+    rndrhmcl.1 $e |- ( ph -> ran F =/= { .0. } ) $.
+    $( The range of a (nontrivial) division ring homomorphism is a division
+       ring.  TODO:  The use of ~ 01eq0ring indicates that ~ df-nzr should come
+       before ~ df-drng .  (Contributed by SN, 17-Feb-2025.) $)
+    rndrhmcl $p |- ( ph -> R e. DivRing ) $=
+      ( va wcel cfv wceq wa eqid wne syl adantr vx crg cui cbs c0g csn cdif cdr
+      crh co crn csubrg rnrhmsubrg subrgring 3syl cv unitcl adantl cur cxp rhmf
+      wn wf rhmrcl2 simpr subrg1 eqcomd 3eqtr3rd 01eq0ring syl2an2r feq3d mpbid
+      subrg0 fvexi fconst2 sylib wfn c0 wb ffnd drngringd ringidcl ne0d fconst5
+      mteqand eleq1 biimpac 0unit biimpa syl2an anassrs nelsn eldifd ressbasss2
+      eldifi sselid fvelrnb simprr wss ssidd resrhm2b syl2anc ad2antrr eldifsni
+      wrex simprl ad2antlr fveq2d simplrr rhmghm ghmid eqtr4d ad3antrrr 3eqtr3d
+      cghm drngunit mpbir2and elrhmunit eqeltrrd rexlimddv impbida eqrdv isdrng
+      sylanbrc ) ABUBMZBUCNZBUDNZBUENZUFZUGZOBUHMACDEUIUJMZCUKZEULNMZYEICDEUMZY
+      LEBGUNUOZAUAYFYJAUAUPZYFMZYPYJMZAYQPZYPYGYIYQYPYGMAYGBYFYPYGQZYFQZUQURYSY
+      PYHRZYPYIMVBYSYPYHBUSNZYHAUUCYHRYQAUUCYHYLFUFZKAUUCYHOZPZCDUDNZUUDUTOZYLU
+      UDOZUUFUUGUUDCVCZUUHUUFUUGEUDNZCVCZUUJAUULUUEAYKUULIUUGUUKDECUUGQZUUKQZVA
+      SZTUUFUUKUUDCUUGAEUBMZUUEFEUSNZOUUKUUDOAYKUUPIDECVDSUUFUUCYHUUQFAUUEVEAUU
+      CUUQOZUUEAYMUURAYKYMIYNSZYMUUQUUCYLEBUUQGUUQQZVFVGSTAYHFOZUUEAYMUVAUUSYMF
+      YHYLEBFGHVMVGSZTVHUUKEUUQFUUNHUUTVIVJVKVLUUGFCFEUEHVNVOVPACUUGVQZUUEUUGVR
+      RZUUHUUIVSAUUGUUKCUUOVTZAUVDUUEAUUGDUSNZADUBMUVFUUGMADJWAUUGDUVFUUMUVFQWB
+      SWCTUUGFCWDVJVLWETAYQYPYHOZUUEAYEYHYFMZUUEYQUVGPYOUVGYQUVHYPYHYFWFWGYEUVH
+      UUEBYFUUCYHUUAYHQZUUCQWHWIWJWKWEYPYHWLSWMAYRPZLUPZCNZYPOZYQLUUGAUVCYPYLMZ
+      UVMLUUGXEZYRUVEYRYGYLYPYLBEGWNYPYGYIWOWPUVCUVNUVOLUUGYPCWQWIWJUVJUVKUUGMZ
+      UVMPZPZUVLYPYFUVJUVPUVMWRUVRCDBUIUJMZUVKDUCNZMZUVLYFMAUVSYRUVQAYKUVSIAYMY
+      LYLWSYKUVSVSUUSAYLWTDEBCYLGXAXBVLXCUVRUWAUVPUVKDUENZRZUVJUVPUVMXFUVRUVKUW
+      BYPYHYRUUBAUVQYPYGYHXDXGUVRUVKUWBOZPZUVLUWBCNZYPYHUWEUVKUWBCUVRUWDVEXHUVJ
+      UVPUVMUWDXIAUWFYHOYRUVQUWDAUWFFYHAYKCDEXOUJMUWFFOIDECXJDECUWBFUWBQZHXKUOU
+      VBXLXMXNWEUVRDUHMZUWAUVPUWCPVSAUWHYRUVQJXCUUGDUVTUVKUWBUUMUVTQUWGXPSXQUVK
+      DBCXRXBXSXTYAYBYGBYFYHYTUUAUVIYCYD $.
+  $}
+
+  ${
+    $d R f $.  $d S f $.
+    $( A ring isomorphism from a division ring goes to a division ring.
+       (Contributed by SN, 18-Feb-2025.) $)
+    ricdrng1 $p |- ( ( R ~=r S /\ R e. DivRing ) -> S e. DivRing ) $=
```

Comment:
Maybe rather "_maps a division ring to a division ring_" ?

(I'm not a native speaker though, I'm not good at comments, so better check with others!)